third_party/libtock-drivers/src/usb_ctap_hid.rs

```diff
@@ -329,7 +330,7 @@ fn send_or_recv_with_timeout_detail(
         #[cfg(feature = "verbose_usb")]
         writeln!(Console::new(), "Cancelling USB transaction due to timeout").unwrap();
         let result_code =
-            unsafe { syscalls::raw::command(DRIVER_NUMBER, command_nr::CANCEL, 0, 0) };
+            unsafe { syscalls::raw::command(DRIVER_NUMBER, command_nr::CANCEL, endpoint as usize, 0) };
```

Comment:
Why does `CANCEL` need an endpoint? There are 2 invocations in this file, and one gets an endpoint, the other doesn't. So I guess we don't need that here.

Reply:
In this flow we have the endpoint that is being used so I thought it best to only cancel that one. Without sending the endpoint number, all endpoints have to be cancelled.
But if you think it is fine to always cancel I can do this. LMK

Reply:
Cancel per endpoint is fine. I thought this must be unintentional that the two `CANCEL` behave differently, but makes sense. We shouldn't need to cancel the other endpoint really.